wdio.conf.js

```diff
@@ -74,6 +74,7 @@ exports.config = {
             './e2e/wdio/platform/**/value-help-dialog.e2e-spec.ts',
             './e2e/wdio/platform/**/time-picker.e2e-spec.ts',
             './e2e/wdio/platform/**/select.e2e-spec.ts',
+            './e2e/wdio/platform/**/upload-collection.e2e-spec.ts',
```

Comment:
need to rebase to the latest main and add spec to `wdio-cron.conf.js` platformB instead of `wdio.conf.js`